ui/components/app/hold-to-reveal/index.scss

```diff
@@ -0,0 +1,41 @@
+.btn--rounded.btn-primary {
+  width: 190px;
+  height: 40px;
+  padding: 6px 13px 6px 9px;
+
+  &:active {
+    background-color: var(--Blue-500) !important;
+  }
+}
+
+.button__icon {
+  margin-inline-end: 11px !important;
+}
+
+.main-box {
+  position: relative;
+
+  &__circular-progress-secondary {
+    z-index: 1;
+    color: var(--ui-white);
+  }
+
+  &__circular-progress-primary {
+    color: rgba(0, 0, 0, 0.3);
+  }
+
+  &__image-box {
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    position: absolute;
+  }
+
+  &__icon {
+    width: 7.88px;
+    height: 9px;
+    left: calc(50% - 7.88px/2 - 72.06px);
+    top: calc(50% - 9px/2 - 0.5px);
```

Comment:
These styles don't seem to do anything and they are using some pretty funky values? I think you can remove them